### PR TITLE
Enable `settingsSync` and `newSettingsStorage` flags

### PR DESCRIPTION
**Pull request by bjtitus**
Please include a summary of the changes and the related issue.

## To test

* Build and run from new install
* Visit the Beta Features screen
* Ensure `settingsSync` and `newSettingsStorage` flags are enabled

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/Automattic/pocket-casts-ios/milestone/44">7.61 ❄️</a>. The due date for this milestone has already passed.<br />
Please assign it to a milestone with a later deadline or check whether the release for this milestone has already been finished.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift [hunk 1]
```diff
@@ -93,7 +93,7 @@ public enum FeatureFlag: String, CaseIterable {
     }
 
     private var shouldEnableSyncedSettings: Bool {
-        false
+        true
```
Comment (leandroalonso) [inline Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift]: Shall we do like we did before? https://github.com/Automattic/pocket-casts-ios/commit/2858b76ce31c2c97314a4a5d01861fb166e4c716

**Review by leandroalonso — APPROVED** [review]
Left a comment about changing the code to what we did previously, not blocking though and I don't feel strong about it.

File: Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift (hunk 1 above)
Comment (bjtitus) [reply Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift]: Yeah, sure. It provides a backstop against a future release, I guess.

**Pull request merged**